iFixit/react-commerce
# Add secondary navbar

Adds a secondary navbar with breadcrumb. Don't pay attention to the breadcrumb items, they will have correct data once the updated breadcrumb metafield is ready.

## QA

1. Visit Vercel preview
2. Go to `/Products/pro-tech-toolkit`
3. Verify that the secondary navbar is present
